tests/library/gdb/tests/test_command_setrlimit.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+from unittest.mock import patch
+
+import pwndbg.commands.setrlimit as sr
+
+
+def test_setrlimit_unknown_resource(capfd):
+    with patch.object(sr, "_invoke_setrlimit") as mock_invoke:
+        sr.setrlimit("unknown", "1")
+        mock_invoke.assert_not_called()
+
+    out, _ = capfd.readouterr()
+    assert "Unknown resource 'unknown'" in out
```

Comment:
Can't we just do `gdb.execute("setrlimit unknown 1", to_string=True) == "Unknown resource 'unknown'"` or something like that instead of doing the mocks etc?

Reply:
I mean, instead of using [the `capfd` fixture across the tests here](https://docs.pytest.org/en/stable/reference/reference.html#std-fixture-capfd)

I think we never use that anywhere heh